agenta-web/src/pages/apps/[app_name]/testsets/new/upload/index.tsx

```diff
@@ -70,27 +72,67 @@ export default function AddANewTestset() {
+                        setUploadType("JSON")
+                    }}
+                >
+                    json
+                </Button>
+            </div>
             <Space direction="vertical" style={{width: "50%"}}>
                 <Alert
                     message="File format"
                     description={
                         <>
-                            The test set should be in CSV format with the following requirements:
-                            <br />
-                            1. Comma separated values
-                            <br />
-                            2. The first row should contain the headers
-                            <br />
-                            <br />
-                            Here is an example of a valid CSV file:
-                            <br />
-                            <br />
-                            recipe_name,correct_answer
-                            <br />
-                            Chicken Parmesan,Chicken
-                            <br />
-                            "a, special, recipe",Beef
+                            The test set should be in {uploadType} format with the following
+                            requirements:
                             <br />
+                            {uploadType == "CSV" && (
+                                <>
+                                    1. Comma separated values
+                                    <br />
+                                    2. The first row should contain the headers
+                                    <br />
+                                    <br />
+                                    Here is an example of a valid CSV file:
+                                    <br />
+                                    <br />
+                                    recipe_name,correct_answer
+                                    <br />
+                                    Chicken Parmesan,Chicken
+                                    <br />
+                                    "a, special, recipe",Beef
+                                    <br />
+                                </>
+                            )}
+                            {uploadType == "JSON" && (
+                                <>
+                                    1. A json file with an array of rows
+                                    <br />
+                                    2. Each row in the array should be an object
+                                    <br />
+                                    of column header name as key and row data as value
+                                    <br />
+                                    <br />
+                                    Here is an example of a valid JSON file:
+                                    <br />
+                                    <br />
+                                    {`[{ recipe_name : "Chicken Parmesan" , correct_answer : "Chicken" },`}
```

Comment:
This is not valid JSON. 

In JSON, keys (property names) should be wrapped in double quotes (`"`). Here's the corrected version:

```json
[
  {
    "recipe_name": "Chicken Parmesan",
    "correct_answer": "Chicken"
  },
  {
    "recipe_name": "a, special, recipe",
    "correct_answer": "Beef"
  }
]
```

Reply:
great, Fixed!